frontend/www/js/omegaup/components/login/Signup.vue

```diff
@@ -84,16 +114,8 @@
                 data-signup-submit
                 class="btn btn-primary form-control"
                 name="sign_up"
-                @click.prevent="
-                  $emit(
-                    'register-and-login',
-                    username,
-                    email,
-                    password,
-                    passwordConfirmation,
-                    recaptchaResponse,
-                  )
-                "
+                :disabled="!birthDate"
```

Comment:
```suggestion
                :disabled="!birthDate || !privacyPolicyAccepted"
```

oops, we never checked for the privacy policy!